src/howler.js

```diff
@@ -0,0 +1,7 @@
+var Howler
+
+if (typeof window !== 'undefined') {
+  Howler = require('howler')
+}
+
+module.exports = Howler
```

Comment:
Alright, just live it there. I have a hard time dealing with es6 and requirejs import/export.